browser/model/virtualbox.js

```diff
@@ -16,6 +16,7 @@ class VirtualBoxInstall extends InstallableItem {
     this.minimumVersion = version;
     this.maximumVersion = '5.2.0';
     this.revision = revision;
+    this.targetDir = "";
```

Comment:
You don't need new class field, see my comments below. Just use location property from 'install' option as it is explained below.